lib/inquisition/outputter/doc.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'html/template'
+require_relative 'documentation/doc_view_helper'
+
+module Inquisition
+  module Outputter
+    class Doc
+      Outputter.declare(self, :stop)
+
+      def initialize(_output); end
+
+      def stop(issues)
+        generate_doc(issues)
+      end
+
+      def generate_doc(_issues)
+        documentation = HTML::Template.new('documentation_word').render(Documentation::DocViewHelper.new)
+        File.open(path_to_word_file, 'w') { |file| file.write(documentation) }
+      end
+
+      private
+
+      def path_to_word_file
```

Comment:
Methods: `path_to_word_file`, `define_file_name`, `project_file_name`, `date_file_name` -  doesn't depend on instance state, maybe move it to another class? What do you think?